getting-started/models/crecto/README.md

```diff
@@ -0,0 +1,313 @@
+# Crecto
```

Comment:
This is great!

Reply:
Yeah, I agree.  I just copied the readme for now.

Reply:
@fridgerator maybe you can help with this?  I haven't actually ever used cretco.  :-)